bouncerscript/test/test_work_dir/task.json

```diff
@@ -0,0 +1,35 @@
+{
+    "provisionerId": "meh",
+    "workerType": "workertype",
+    "schedulerId": "task-graph-scheduler",
+    "taskGroupId": "some",
+    "routes": [],
+    "retries": 5,
+    "created": "2015-05-08T16:15:58.903Z",
+    "deadline": "2015-05-08T18:15:59.010Z",
+    "expires": "2016-05-08T18:15:59.010Z",
+    "dependencies": ["eSzfNqMZT_mSiQQXu8hyqg"],
+    "scopes": [
+        "project:releng:bouncer:server:staging",
+        "project:releng:bouncer:action:submission"
+    ],
+    "payload": {
+        "locales": ["en-US", "ro"],
+        "submission_entries": {
+            "Firefox-99999": {
```

Comment:
Nit: This might be good to have a second entry in the payload. This would ensure we do iterate on them